src/main/java/uk/gov/companieshouse/api/accounts/model/ixbrl/company/Company.java

```diff
@@ -24,4 +34,20 @@ public String getCompanyName() {
     public void setCompanyName(String companyName) {
         this.companyName = companyName;
     }
+    
+    private void getCompanyJurisdiction(String companyNumber) {
```

Comment:
Bit of a nitpick but I think this should be a set method. Still just setting jurisdiction here.